### DISCUSSION TITLE
Adding my domain using name.com

### DISCUSSION THREAD

**Discussion by Beefy-py** [vercel/community#1419] [Help] [answered]
### Summary

I got a free domain from [name](https://www.name.com/) for a year.
Now I need to add it to my Vercel project but I'm having trouble with it.
I set  the following two name servers in name.com ```
ns1.vercel-dns.com
ns2.vercel-dns.com```
When I check my projects domain settings it says valid configuration for both domains but then it refreshes and says invalid.

### Additional information

https://kennyhoft.live should work but now my site is on https://kenny-hoft.vercel.com

**Comment by amyegan** [comment, marked as answer]
Hi Kenny! Are you still having trouble with this? Both https://www.kennyhoft.live/ and https://kenny-hoft.vercel.app/ seem to let me reach your site without any problems now.

**Reply by Beefy-py** [reply]
Yes,
They are fine now. I should have waited for a couple more hours before asking.
; )
